src/scheduler/FunctionCallClient.cpp

```diff
@@ -1,11 +1,92 @@
+#include <faabric/proto/faabric.pb.h>
 #include <faabric/scheduler/FunctionCallClient.h>
 
 #include <grpcpp/create_channel.h>
 #include <grpcpp/security/credentials.h>
 
 #include <faabric/proto/macros.h>
+#include <faabric/util/queue.h>
+#include <faabric/util/testing.h>
 
 namespace faabric::scheduler {
+
+// -----------------------------------
+// Mocking
+// -----------------------------------
```

Comment:
This whole "mocking" section is for testing calls made to other hosts. It's easier than trying to run two servers on the same host.